server/pulp/server/db/migrations/0024_distributor_last_updated.py

```diff
@@ -15,4 +16,9 @@ def migrate(*args, **kwargs):
             distributor[key] = distributor['last_publish']
```

Comment:
if i omit continue statement and the key with its value will be already present in the collection, then instead of be left untouched it will be changed, which is what we don't want.